.github/workflows/release.yml

```diff
@@ -80,6 +79,22 @@ jobs:
           otp: ${{ matrix.otp }}
           build_docs: ${{ matrix.build_docs }}
 
+      - name: "Sign files with Trusted Signing"
```

Comment:
A side-effect of moving this here is that it will now run for the latest tags. If this is not desirable, I'll have to add a condition. (`if: github.ref_type == 'branch'`)